AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/README.md

```diff
@@ -1,139 +1,121 @@
+
+>**Note**: Intel® Extension for PyTorch (IPEX) can be installed via the Intel® AI Tools Offline Installer or via the Intel AI Tools Selector. For more information on the optimizations as well as performance data, see [*Intel and Facebook* collaborate to boost PyTorch* CPU performance*](http://software.intel.com/en-us/articles/intel-and-facebook-collaborate-to-boost-pytorch-cpu-performance).
+
+>
+>Find more examples in the [*Examples*](https://intel.github.io/intel-extension-for-pytorch/cpu/latest/tutorials/examples.html) topic of the [*Intel® Extension for PyTorch (IPEX) Documentation*](https://intel.github.io/intel-extension-for-pytorch).
+
+
+## Prerequisites
+
+| Optimized for        | Description
+|:---                  |:---
+| OS                   | Ubuntu* 22.04
+| Hardware             | Intel® Xeon® scalable processor family <br> Intel® Data Center GPUs
+| Software             | Intel® Extension for PyTorch (IPEX)
+
+> **Note**: AI and Analytics samples are validated on AI Tools Offline Installer. For the full list of validated platforms refer to [Platform Validation](https://github.com/oneapi-src/oneAPI-samples/tree/master?tab=readme-ov-file#platform-validation).
+
+## Key Implementation Details
+
+The sample uses pretrained model provided by Intel and published as part of [Intel AI Reference Models](https://github.com/IntelAI/models). The example also illustrates how to utilize TensorFlow* and Intel® Math Kernel Library (Intel® MKL) runtime settings to maximize CPU performance on ResNet50 workload.
+
+
+- The Jupyter Notebook, `ResNet50_Inference.ipynb`, is implemented for both CPU and GPU using Intel® Extension for PyTorch (IPEX).
+- The `Intel_Extension_For_PyTorch_Hello_World.py` script is implemented for CPU using the Python language.
+- You must export the environment variable `DNNL_VERBOSE=1` to display the deep learning primitives trace during execution.
+
+> **Note**: The test dataset is inherited from `torch.utils.data.Dataset`, and the model is inherited from `torch.nn.Module`.
+
+## Environment Setup
+You will need to download and install the following toolkits, tools, and components to use the sample.
+
+**1. Get Intel® AI Tools**
+
+Required AI Tools: <https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html>
+<br>If you have not already, select and install these Tools via [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html). AI and Analytics samples are validated on AI Tools Offline Installer. It is recommended to select Offline Installer option in AI Tools Selector.
+
+
+
+## Run the Sample
+>**Note**: Before running the sample, make sure [Environment Setup](https://github.com/ZhaoqiongZ/oneAPI-samples/tree/development/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted#environment-setup) is completed.
```

Comment:
@ZhaoqiongZ This is pointing to your fork. Can you update it and also use a relative path?

Reply:
Hi @jimmytwei , fix the path, thanks!